.gitignore

```diff
@@ -11,11 +11,12 @@ notes/*
 examples/quickstart/minio/*
 *.bin
 *.meta
+.vscode/*
```

Comment:
This should stay as it has project settings in it.

Reply:
hmm... do you mind if I add my launch.json then?